src/components/annotations/draw.js

```diff
@@ -487,15 +487,15 @@ function drawRaw(gd, options, index, subplotId, xa, ya) {
             // to get the parity of the number of intersections.
             if(edges.reduce(function(a, x) {
                 return a ^
-                    !!lineIntersect(headX, headY, headX + 1e6, headY + 1e6,
+                    !!Lib.segmentsIntersect(headX, headY, headX + 1e6, headY + 1e6,
```

Comment:
Not part of this PR and I'm sure no meaningful performance hit, but is `reduce` necessary here? Seems like

```js
for (i = 0; i < edges.length; i++) {
  if (!Lib.segmentsIntersect(...)) return;
}
```

would be more concise and readable and would bail out early when it can.